frontend/src/components/dashboard/aliases/BlockLevelSlider.module.scss

```diff
@@ -29,11 +29,11 @@ $stopLabelHeight: 2rem;
 
   .control {
     width: 100%;
-    max-width: $content-sm;
+    max-width: $content-xs;
     display: flex;
     flex-direction: column;
     align-items: center;
-    padding: 0 $spacing-lg;
+    padding-left: $spacing-md;
```

Comment:
Nitpicky, but shouldn't we have this padding on both sides? It looks a little bit off on small screens for me now, I think:

![image](https://user-images.githubusercontent.com/4251/164702742-f160a680-599a-4215-9ba7-815692feaef2.png)